### ISSUE TITLE
lint-stage configuration

### ISSUE THREAD

**Issue by weirdpattern** [bradzacher/eslint-plugin-typescript#190]
**Repro**
This has to do with the lint-staged hook in the repository and the fact that is only reporting style issues, but not actually formatting the code.

```JSON
{
  "lint-staged": {
        "*.js": [
            "yarn format-check",
            "git add"
        ]
    }
}
```

**Expected Result**
Code is formatted and staged

**Actual Result**
Code format issues are reported, but not formatted... 

**Additional Info**
I think we should use `yarn format` instead of `yarn format-check`

```JSON
{
  "lint-staged": {
        "*.js": [
            "yarn format",
            "git add"
        ]
    }
}
```

**Versions**

| package                    | version |
| -------------------------- | ------- |
| `eslint-plugin-typescript` | `0.14.0` |
| `typescript-eslint-parser` | `17.0.1` |
| `typescript`               | `2.9` |
